packages/mds-transactions-service/repository/index.ts

```diff
@@ -22,6 +24,19 @@ import { TransactionOperationEntity } from './entities/operation-entity'
 import { TransactionStatusEntity } from './entities/status-entity'
 import migrations from './migrations'
 
+const { validate: validateTransactionSearchParams } = schemaValidator<TransactionSearchParams>(
+  Joi.object<TransactionSearchParams>()
+    .keys({
+      provider_id: Joi.string().uuid(),
+      start_timestamp: Joi.number().integer(),
+      end_timestamp: Joi.number().integer(),
+      before: Joi.string(),
```

Comment:
Shouldn't all of thse be optional, since they are all optional in `TransactionSeachParams`?

Reply:
@janedotx I believe that by default Joi schemas are implicitly `optional`, you need to specify `.required()` on each property if you want them to be required.